app/config/packages/doctrine.yaml

```diff
@@ -0,0 +1,28 @@
+parameters:
+    database_host:     127.0.0.1
```

Comment:
IMO, it's not the good host. 
And it's probably not the good place to set this config parameters.